docs/reference/api/platform/Error.md

```diff
@@ -110,9 +110,11 @@ This is equivalent to defining an error status with `MODULE_UNKNOWN`. However, u
 
 ### Error history
 
-Error handling implementation in Mbed OS also keeps track of previous errors in the system. This feature is called **Error history** and is configurable using the configuration value **MBED_CONF_ERROR_HIST_SIZE**.
+Error handling implementation in Mbed OS keeps track of previous errors in the system. This feature is called **Error history** and is configurable using the configuration value **MBED_CONF_ERROR_HIST_SIZE**.
 
-`MBED_CONF_ERROR_HIST_SIZE` configures the number of previous errors the system keeps in its error history. You can disable the history feature by setting `MBED_CONF_ERROR_HIST_SIZE` to 0. By default, it keeps track of the past four errors. Whether error history is enabled or not, the system always records the first and last errors that happened in the system. See the below link to learn more about the APIs related to error history:
+`MBED_CONF_ERROR_HIST_SIZE` configures the number of previous errors the system keeps in its error history. You can disable the error history feature by defining `MBED_CONF_ERROR_HIST_DISABLED`. By default, it keeps track of the past four errors. Whether error history is enabled or not, the system always records the first and last errors that happened in the system. APIs are provided to retrieve errors or warnings from the **Error history** and the first and last errors. Note that in most cases calling **MBED_ERROR()/MBED_ERROR1()** halts the system, and thus error history APIs retrieves the warnings, unless you are calling these APIs from the error hook(see the [error hook for applications](#error-hook-for-applications) section below) function.
```

Comment:
In offline discussions with @SenRamakri we talked a bit about adding these configuration options to the config system. Not a blocking comment for this PR, just wanted to mention the wording here and in other instances where the configuration options are described should be changed to referencing the config options system when it's added.

Reply:
@kegilbert - Yes, I'll be working on capturing this in platform/mbed_lib.json. It will be a future PR on mbed-os side.